javascript-loops-practice/find-index.js

```diff
@@ -0,0 +1,14 @@
+/* exported findIndex */
+function findIndex(array, value) {
+  let num = 0;
+  for (let i = 0; i < array.length; i++) {
+    if (array[i] === value) {
+      return i;
+    } else {
+      num = -1;
+    }
+  }
+  return num;
```

Comment:
Instead of returning `num` here, I would just `return -1`. You'll essentially never return `num` at it's original value of 0, & you only ever update `num` if the `value` isn't equal to `array[i]`, so I would just let the whole loop run, and if your `if` statement never hits, then you would just `return -1` directly.